networkx/conftest.py

```diff
@@ -54,16 +54,26 @@ def set_warnings():
         message="This will return a generator in 3.0*",
     )
     warnings.filterwarnings(
-        "ignore", category=DeprecationWarning, message="betweenness_centrality_source*",
+        "ignore",
```

Comment:
Did you mean to reformat this?

Reply:
No, it must've gotten picked up by `black` running via the pre-commit hooks. I can undo the reformatting not related to the changes I've made

Reply:
Hmmm, sounds familiar :). We definitely should have a `pyproject.toml` file in the repo.

Reply:
Or something like https://github.com/networkx/networkx/pull/4235.

Reply:
Or maybe just forget about style until we want to focus on it -- like before each release?

Reply:
I may have figured out why this happening.  I looked into how the ``pyproject.toml`` file lookup goes and I don't think you and Eric were overwriting black's default.  I think you were using a different version of black, which has different defaults.  It appears that new versions of black wrap lists that end in a comma, while the old version didn't.

@rossbar, @ericmjl  What version of black are you using?  Did you upgraded to ``20.8b0`` or ``20.8b1`` already.  We were using ``black==19.10b0`` before #4235.  Once I merge that and you rebase on master, I suspect you won't have this problem any more.

If this is correct, then adding a ``pyproject.toml`` won't currently solve this problem.  There is an issue about this, but it doesn't look like anything has been implemented yet:
https://github.com/psf/black/issues/1246

For now, I think we may all need to use the same version of black and be careful to only upgrade the version when the project starts using the new version.

Reply:
None of the linting tools are installed directly in my venv (`black` doesn't show up in `pip freeze` or `pip list`) - I rely on the precommit hooks to handle it. From the `.pre-commit-config.yaml`, it looks like the `stable` version of black is being grabbed each time:

```yaml
repos:                                                                          
-   repo: https://github.com/ambv/black                                         
    rev: stable                                                                 
    hooks:                                                                      
    - id: black
```

The `black` version changing would explain why the formatting defaults aren't consistent.

Reply:
@rossbar helped me track the problem down.  It looks like it a problem w/  ``.pre-commit-config.yaml``.

I originally had:
```
repos:
repos:
-   repo: https://github.com/ambv/black
-   repo: https://github.com/ambv/black
    rev: stable
    rev: 20.8b1
    hooks:
    hooks:
    - id: black

@rossbar helped me track the problem down.  It looks like it a problem w/  ``.pre-commit-config.yaml``.

I originally had:
```
repos:
-   repo: https://github.com/ambv/black
    rev: stable
    hooks:
    - id: black
```
I just changed it to:
```
repos:
-   repo: https://github.com/ambv/black
    rev: 20.8b1
    hooks:
    - id: black
```
in #4235.

Going forward, I will just need to remember to update both ``.pre-commit-config.yaml`` and ``requirements/developer.txt`` when we upgrade to a new version of black.